src/Tizen.Core/Tizen.Core/Task.cs

```diff
@@ -674,7 +718,10 @@ public void Run()
         public void Quit()
         {
             Interop.LibTizenCore.ErrorCode error = Interop.LibTizenCore.TizenCore.TaskQuit(_handle);
-            TCoreErrorFactory.CheckAndThrownException(error, "Failed to quit task");
+            if (error != Interop.LibTizenCore.ErrorCode.None)
+            {
+                throw new InvalidOperationException("Failed to run task");
```

Comment:
It needs to be modified.
```suggestion
                throw new InvalidOperationException("Failed to quit task");
```

Reply:
Done